mozilla/foundation.mozilla.org
# Embedded Audio Player for blog pages

Closes #6677 

**Link to sample test page**: https://foundation-s-6677-embed-93yzs4.herokuapp.com/en/blog/initial-test-blog-post-with-fixed-title/

Steps to test:

1. Visit https://foundation-s-6677-embed-93yzs4.herokuapp.com/en/blog/initial-test-blog-post-with-fixed-title/, and scroll down to find the audio player. 
2. The play button, progress bar, and time should all load correctly.
3. Press the play button, music should begin to play. 

Note: Since migrations are needed, the Wagtail_CI test will fail. Once approved, I will pull latest migrations and create mine to ensure they are up to date. 


## Checklist

**Changes in Models:**
